test/cpp/test_tensor.cpp

```diff
@@ -10,7 +10,12 @@
 namespace torch_xla {
 namespace cpp_test {
 
-TEST(TensorTest, TestAdd) {
+class TensorTest : public ::testing::Test {
```

Comment:
I think this is a generic base class we can use for tests, so I'd put it in a dedicated cpp/h files.

Reply:
Ok, then inherit something like `TensorOperationsTest` from it?

Reply:
I am fine with the TensorTest class name.